### PR TITLE
submitting assignment 1

### PR DESCRIPTION
**Pull request by nipunpuri**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
submitting assignment 1
## What did you learn from the changes you have made?
train model, tune and how to handle overfitting
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
na
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
overfitting
## How were these changes tested?
in the google collab
## A reference to a related issue in your repository (if applicable)
no
## Checklist
- [x] I can confirm that my changes are working as intended

### CODE REVIEW

File: 02_activities/assignments/assignment_1.ipynb
Comment (tianyi21) [inline 02_activities/assignments/assignment_1.ipynb:1]: - Task 3: When re-train the model, please always reinitialize your model with `model = Sequential() ...` to reset all model parameters in addition to `model.compile()`. Or, you are continuing the training from the previous trained state with a different model setup. You may check this with the accuracy of the first epoch. So here you trained the model for a total `10+5+20=35` epochs

**Review by tianyi21 — APPROVED** [review]
Hi nipun!

Thank you for submitting your DL assignments. Please refer to my comments attached to each file.

Your A1 is now complete.

Thanks,
Tianyi [LS]

File: 01_materials/labs/lab_2.ipynb
Comment (tianyi21) [inline 01_materials/labs/lab_2.ipynb:1]: - Feedforward Multilayer: Since we're using `NLL` loss function, as suggested in the code template, please apply `sigmoid` activation function in `forward_output()`. However, `softmax` is also an acceptable solution.

**Review by tianyi21 — APPROVED** [review]
Hi nipun!

Thank you for submitting your DL assignments.

All your DL assignments have been completed. Thank you for your participation, and I wish you all the best for your future!

Thanks,
Tianyi [LS]
